src/auslib/web/admin/base.py

```diff
@@ -93,6 +93,7 @@ def complete_request(response):
             request.transaction.rollback()
         else:
             request.transaction.commit()
+        request.transaction.close()
```

Comment:
Might need to do this in a try/finally block so we still close the connection when commit fails?

Reply:
Updated.